modules/acquia_cms_common/acquia_cms_common.module

```diff
@@ -381,6 +383,28 @@ function acquia_cms_common_modules_installed($modules, $is_syncing) {
       // Set telemetry to be TRUE on install of acquia_connector.
       \Drupal::state()->set('acquia_connector.telemetry.opted', TRUE);
     }
+    if (in_array('entity_clone', $modules)) {
+      _acquia_cms_common_rewrite_configuration('entity_clone.settings', 'acquia_cms_common');
+    }
+    if (in_array('smart_trim', $modules)) {
+      // List of entity view display configuration to specific modules.
+      $modules_suffix = [
+        'article',
+        'event',
+        'page',
+        'person',
+        'place',
+      ];
+      // Iterate optional configs to be included for entity view display.
+      foreach ($modules_suffix as $suffix) {
+        $acms_module = "acquia_cms_$suffix";
+        // Check if module exists.
+        if ($module_handler->moduleExists($acms_module)) {
+          _acquia_cms_common_rewrite_configuration("core.entity_view_display.node.$suffix.teaser", $acms_module);
+          _acquia_cms_common_rewrite_configuration("core.entity_view_display.node.$suffix.search_results", $acms_module);
+        }
+      }
+    }
```

Comment:
@apathak18, I believe it would be more appropriate to relocate this code from the .install file to a helper or service file and then invoke the method from there. What are your thoughts on this, @vishalkhode1?

Reply:
But it's in .module file

Reply:
updated the file extension in comment.